src/components/LibraryItem.tsx

```diff
@@ -245,4 +246,18 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
             libraryContainer.raiseEvent("itemClicked", this.props.data.contextData);
         }
     }
+
+    onLibraryItemHoveredOn() {
+        let libraryContainer = this.props.libraryContainer;
+        if (this.props.data.childItems.length == 0) {
+            libraryContainer.raiseEvent("itemHoveredOn", this.props.data.contextData);
```

Comment:
This looks more like something that should be documented: "If you want to respond to something, then you need to register some event".